src/function/aggregate/sorted_aggregate_function.cpp

```diff
@@ -189,7 +189,9 @@ struct SortedAggregateState {
 		}
 
 		if (count > CHUNK_CAPACITY && !ordering) {
-			InitializeChunks(order_bind);
+			if (CHUNK_CAPACITY < LIST_CAPACITY) {
```

Comment:
I don't think this works because `FlushLinkedLists` still assumes that `LIST_CAPACITY` <= `CHUNK_CAPACITY`. I think it makes more sense to fix the constants so that `LIST_CAPACITY = min(LIST_CAPACITY, CHUNK_CAPACITY)`

Reply:
Hmm okay, fair enough, I'm not knowledge enough on the code to reason what LIST_CAPACITY should be relative to CHUNK_CAPACITY, happy to make that change if you say so 👍